### DISCUSSION TITLE
SOC timestamp different from ElasticSearch

### DISCUSSION THREAD

**Discussion by i-and-me** [Security-Onion-Solutions/securityonion#9645] [Unsupported Versions]
Hi all,

Recently I added a fortinet module from filebeat. Works good, but The timezone wasn't correct, so I edited the pipeline do to so.
Worked fine too.

But now, the timestamp from SOC and ElasticSearch is different, See screenshot
![image](https://user-images.githubusercontent.com/123649410/214863898-16062105-9dc1-4d31-a4c0-ffbac4d08821.png)

The @ timestamp field on log is the correct one, and the SOC Timestamp is wrong

Edit 1: The pipeline came back to ingest in wrong timezone

**Comment by InfosecGoon** [comment]
I'm seeing something similar on my own Fortinet logs -- in my case, the difference is UTC vs. local time zone. Is it the same for you?
